lib/ResqueStatsD.php

```diff
@@ -246,15 +246,11 @@ private static function joinTags($tags) {
         return '|#' . implode(',', $joinedTags);
     }
 
-    private static function getJobClass($jobOrClass, $args = null) {
-        $className = '';
-        if ($args) {
-            $className = $jobOrClass;
-        } else {
-            $args = $jobOrClass->payload['args'];
-            $className = $jobOrClass->payload['class'];
-        }
+    private static function getJobClassFromJob($job) {
+        return self::getJobClass($job->payload['class'], $job->getArguments());
+    }
 
+    private static function getJobClass($className, $args = null) {
         if ($className === 'Job' && isset($args['callable'])) {
```

Comment:
Let's add a check for `is_array($args['callable'])` since we're expecting array inside the if condition:
```
if ($className === 'Job' && isset($args['callable']) && is_array($args['callable'])) {
```